### PR TITLE
Clean ups

### LINKED ISSUE #115: print statements in test files
There are many `print()` statements on several test files.  I guess they were added for debugging, but I wonder if it is time to remove them.

@ChrisBarker-NOAA I can send a PR for that.  I just want to her from you first.  Maybe there is another reason for those print statements.

### PR DESCRIPTION
**Pull request by ocefpaf**
Closes #115.

If any of those print statements where part of a human inspection test we should convert it to a `unittest`.

### CODE REVIEW

**Comment by ocefpaf** [comment]
@ChrisBarker-NOAA I please let me know if any of those should be converted into a `unittest`.  I will do so following you instructions, but as I am still getting familiarizar with the code I cannot see any obvious one.

File: test/sample_script.py [hunk 1]
```diff
@@ -1,28 +0,0 @@
...
-# Maybe we don't want to do that.
-print("Loading data: This could take a while...")
-ug = pyugrid.UGrid.from_ncfile(url)
-
-# What's in there?
-print("There are %i nodes" % ug.nodes.shape[0])
-print("There are %i edges" % ug.edges.shape[0])
-print("There are %i faces" % ug.faces.shape[0])
-
-print('The start of the "connectivity array":', ug.faces[:10])
```
Comment (ChrisBarker-NOAA) [inline test/sample_script.py]: this one is pretty pointless I think -- now that we have a few notebook examples. So I say either add soem more to it, bot jsut get rid of it.

it probably shouldn't be in the tests dir in any case.

File: test/test_examples.py [hunk 2]
```diff
@@ -112,8 +112,4 @@ def twenty_one_triangles():
 
 if __name__ == "__main__":
     grid = twenty_one_triangles()
-    print(grid.edges)
```
Comment (ChrisBarker-NOAA) [inline test/test_examples.py]: When I put code in an if __name__ block in a test file, it's so you can play around a bit without dealing with the full-on test runner. That's usually through-away code (as it won't be run by the test runner anyway).

File: test/test_save_as_netcdf.py [hunk 3]
```diff
@@ -30,7 +30,6 @@ def nc_has_variable(ds, var_name):
     if var_name in ds.variables:
         return True
     else:
-        print(var_name, " is not a variable in the Dataset")
```
Comment (ChrisBarker-NOAA) [inline test/test_save_as_netcdf.py]: These prints may be worth keeping -- if a test fails, you may not know which variable, attribute, etc is causing the failure, so this is kind of handy.

**Comment by ChrisBarker-NOAA** [comment]
with the exception of test_save_as_netcdf.py -- this all looks great -- thanks for the cleanup.

File: test/test_examples.py (hunk 2 above)
Comment (ocefpaf) [inline test/test_examples.py]: OK.  I am returning that `print` statement.

File: test/test_save_as_netcdf.py (hunk 3 above)
Comment (ocefpaf) [inline test/test_save_as_netcdf.py]: In a way that is testing the data and not `pyugrid`, right? Or maybe I am missing something.

Comment (ocefpaf) [inline test/test_save_as_netcdf.py]: Yes... I am missing something! It tests the `pyugrid` saved data.

PS: I returned the print statement, but I think `py.test` verbose mode would give you the full `traceback` and that would not be needed.

File: test/sample_script.py (hunk 1 above)
Comment (ocefpaf) [inline test/sample_script.py]: That think that file is @rsignell-usgs' addition.  I vote that it can go.  The notebooks cover that example (and more).

**Comment by ocefpaf** [comment]
@ChrisBarker-NOAA I made the changes, rebsaed, and ready to go for another round of reviews.

**Comment by ChrisBarker-NOAA** [comment]
This all looks great to me -- thanks!

**Comment by ocefpaf** [comment]
Thanks!

**Pull request merged**